src/test/java/io/github/pmckeown/dependencytrack/project/ProjectClientIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package io.github.pmckeown.dependencytrack.project;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.exactly;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.verify;
+import static io.github.pmckeown.dependencytrack.TestResourceConstants.V1_PROJECT_WITHOUT_PAGINATION;
+import static io.github.pmckeown.dependencytrack.TestResourceConstants.V1_PROJECT_WITH_PAGINATION;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.github.pmckeown.dependencytrack.AbstractDependencyTrackMojoTest;
+import io.github.pmckeown.dependencytrack.CommonConfig;
+
+public class ProjectClientIntegrationTest extends AbstractDependencyTrackMojoTest {
```

Comment:
I don't see how this test is different from the existing integration tests that stub out a call to get projects and verifies the response contains the expected amount.

Reply:
Yes I agree it similar. 
The other integration tests were focused on the testing the mojos, and I would have had to piggy back on one or all of those that called ProjectClient.getProjects().  
I felt it was better to create a test solely focused on ProjectClient.